src/server/storage/verifier.js

```diff
@@ -75,6 +88,4 @@ class UserVerifier {
   }
 }
 
-UserVerifier.addStrategy('facebook', FacebookVerificationStrategy)
-
-export default (userRecord, requestPayload, logger) => new UserVerifier(userRecord, requestPayload, logger)
+export default UserVerifier.factory
```

Comment:
i think its missing initialization

Reply:
no, because user verifier isn't a signletone
it's initialized for each request with add user payload, user record pre-created and req.log.
So verifeir.js exports a factory instead of object instance

factory is imported here:
https://github.com/GoodDollar/GoodServer/blob/master/src/server/storage/storageAPI.js#L12

then we're creating instance here once we have user data:
https://github.com/GoodDollar/GoodServer/blob/master/src/server/storage/storageAPI.js#L39